### ISSUE TITLE
Update Kramdown version due to CVE-2020-14001

### ISSUE THREAD

**Issue by DannyBen** [piotrmurach/tty-markdown#24]
Kramdown dependency version should be updated to `~> 2.3.0` due to [CVE-2020-14001](https://github.com/advisories/GHSA-mqm2-cgpr-p4m6)

In addition, the new Kramdown `>= 2.0` is no longer coming bundled with the GFM parser ([ref](https://github.com/gettalong/kramdown/blob/42280631fc748fae952e7bc60058cfbc2129a422/doc/parser/gfm.page)), so it also needs to be brought in.

```diff
-  spec.add_dependency "kramdown",   "~> 1.16.2"
+  spec.add_dependency "kramdown",   "~> 2.3"
+  spec.add_dependency "kramdown-parser-gfm", "~> 1.1"
```

**Comment by piotrmurach** [comment]
Thank you! Please see my comments in the PR. I'm soon going to work on fixing all the issues and bringing `tty-markdown` up to date. I left things half-finished and need to look at everything once again.

**Comment by DannyBen** [comment]
Any estimate as to when this will make it to a released gem? All users are still vulnerable.

**Comment by piotrmurach** [comment]
Released `v0.7.0` with many updates (see [changelog](https://github.com/piotrmurach/tty-markdown/blob/master/CHANGELOG.md)). I've put many hours into this version and hope it's much better experience.

**Comment by DannyBen** [comment]
Thank you.
